rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3FilterHandler.java

```diff
@@ -67,15 +72,41 @@ protected void channelRead0(ChannelHandlerContext ctx, FullHttpRequest request)
         // the request will be inmediately rejected since it won't be parsed as a valid URI
         // and won't work to match an item on rpc.host
         String hostHeader = headers.get(HttpHeaders.Names.HOST);
-        String parsedHeader = parseHostHeader(hostHeader);
 
-        if (!acceptedHosts.contains(parsedHeader)) {
-            logger.debug("Invalid header HOST {}", hostHeader);
-            response = new DefaultHttpResponse(HttpVersion.HTTP_1_1, HttpResponseStatus.BAD_REQUEST);
-            ctx.write(response).addListener(ChannelFutureListener.CLOSE);
+        if (hostHeader.isEmpty()) {
```

Comment:
I'd also check for `null` here (case where host header is not provided)

Reply:
Done